Formula/gst-editing-services.rb

```diff
@@ -25,6 +25,10 @@ class GstEditingServices < Formula
   depends_on "gst-plugins-base"
   depends_on "gstreamer"
 
+  on_linux do
+    depends_on "flex" => :build
```

Comment:
This should be:
`uses_from_macos "flex" => :build`

It will not solve the issue but it's a shorter (and better) syntax.